diff --git a/dist/esm/routes/oauth/callback.mjs b/dist/esm/routes/oauth/callback.mjs
index f77ed76d2737ad004281e941552054233dcaa63c..7c17523019d39e88cddb006d5328f39588c59a37 100644
--- a/dist/esm/routes/oauth/callback.mjs
+++ b/dist/esm/routes/oauth/callback.mjs
@@ -6,6 +6,8 @@ import { Route } from '../../lib/structures/Route.mjs';
 import { HttpCodes } from '../../lib/structures/http/HttpCodes.mjs';
 import { methods } from '../../lib/structures/http/HttpMethods.mjs';
 
+const APP_DOMAIN = process.env.FLY_APP_NAME ? `${process.env.FLY_APP_NAME}.fly.dev` : '127.0.0.1'
+
 var _PluginRoute = class _PluginRoute extends Route {
   constructor(context) {
     super(context, { route: "oauth/callback" });
@@ -35,7 +37,7 @@ var _PluginRoute = class _PluginRoute extends Route {
       refresh: value.refresh_token,
       token: value.access_token
     });
-    response.cookies.add(auth.cookie, token, { maxAge: value.expires_in });
+    response.cookies.add(auth.cookie, token, { maxAge: value.expires_in, httpOnly: false, domain: APP_DOMAIN });
     return response.json(data);
   }
   async fetchAuth(body) {
